scripts/openssl.test

```diff
@@ -467,7 +467,7 @@ IFS=$OIFS #restore separator
 #
 
 # Check for cerificate support in wolfSSL
-wolf_certs=`$WOLFSSL_CLIENT -help 2>&1`
+wolf_certs=`$WOLFSSL_CLIENT -? 2>/dev/null`
```

Comment:
This will properly match now. But will always trigger the failures of the test cases that until #4017 were being skipped.